pages/portfolio.html

```diff
@@ -4,9 +4,51 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <link href="styles/portfolio.css" rel="stylesheet">
+  <title>Programming Portfolio</title>
 </head>
 <body>
-  
+  <header>
+    <h1>Programming Portfolio</h1>
+  </header>
+
+  <nav>
+    <a href="index.html">Home</a>
+    <a href="about.html">About Me</a>
+  </nav>
+
+  <section>
```

Comment:
Even more than being an arbitrary section, this feels like the main content to me, so I might use the main tag here instead.